app/views/expenses/index.slim

```diff
@@ -4,31 +4,35 @@
 
 - else
 
-  table
+
+
+  table.table.table-striped
     thead
       tr
-        th Payment Date
-        th Category
-        th Sub-Category
-        th Description
-        th Supplier
-        th Amount
+        th DATE
```

Comment:
It'd be better to have this text in regular casing and do it in css with text-transform.